RHDEVDOCS-5986: Multiple authentication support in Git resolver (#84201)
openshift/openshift-docs: modules/op-resolver-git-specify-multiple-providers.adoc

> // This module is included in the following assemblies:
> // * create/remote-pipelines-tasks-resolvers.adoc
> 
> // // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
> :_mod-docs-content-type: PROCEDURE

Reviewer: ```suggestion
// This module is included in the following assemblies:
// * create/remote-pipelines-tasks-resolvers.adoc
// * openshift_pipelines/remote-pipelines-tasks-resolvers.adoc

:_mod-docs-content-type: PROCEDURE
```

[id="op-resolver-git-specify-multiple-providers_{context}"]
= Specifying multiple Git providers

You can specify multiple Git providers by passing the unique `configKey` parameter when creating `TaskRun` and `PipelineRun` resources.

If no `configKey` parameter is passed, the default configuration is used. You can also specify default configuration by setting the `configKey` value to `default`.

[WARNING]
====
The `configKey` values with the `.` symbol are not supported. If you try to pass the `configKey` value with the `.` symbol, the `TaskRun` or `PipelineRun` resource where you passed the value fails to run.

> ====
> .Prerequisites

Reviewer: Spacing:
```suggestion
====

.Prerequisites
```

> * Configure multiple Git providers through `Tektonconfig` custom resource. For more information, see "Configuring multiple Git providers".

Reviewer: ```suggestion
* Configure multiple Git providers through the `Tektonconfig` custom resource. For more information, see "Configuring multiple Git providers".
```

.Procedure

* To specify a Git provider, use the following reference format in the `pipelineRef` and `taskRef` spec:
+
[source,yaml]
----
# ...
  resolver: git
  params:
  # ...
  - name: configKey
    value: <your_unique_key> # <1>
# ...
----

> <1> Your unique key that is matching one of the configuration keys, for example, `test1`.

Reviewer: Nit/alternatives:
```suggestion
<1> Your unique key that matches one of the configuration keys, for example, `test1`.
```
or
```suggestion
<1> Your unique key matching one of the configuration keys, for example, `test1`.
```

// This module is included in the following assemblies:
// * create/remote-pipelines-tasks-resolvers.adoc
// * openshift_pipelines/remote-pipelines-tasks-resolvers.adoc

:_mod-docs-content-type: PROCEDURE
[id="op-resolver-git-specify-multiple-providers_{context}"]
= Specifying multiple Git providers

You can specify multiple Git providers by passing the unique `configKey` parameter when creating `TaskRun` and `PipelineRun` resources.

If no `configKey` parameter is passed, the default configuration is used. You can also specify default configuration by setting the `configKey` value to `default`.

[WARNING]
====

> The `configKey` values with the `.` symbol are not supported. If you try to pass the `configKey` value with the `.` symbol, the `TaskRun` or `PipelineRun` resource where you passed the value fails to run.

Reviewer 2: Same note as before.
